scripts/vm/network/security_group.py

```diff
@@ -185,7 +185,7 @@ def destroy_network_rules_for_nic(vm_name, vm_ip, vm_mac, vif, sec_ips):
         logging.debug("Ignoring failure to delete ebtable rules for vm: " + vm_name)
 
 def get_bridge_physdev(brname):
-    physdev = execute("bridge -o link show | awk '/master %s / && !/^[0-9]+: vnet/ {print $2}' | head -1" % brname)
+    physdev = execute("bridge -o link show | awk '/master %s / && !/^[0-9]+: vnet/ {print $2}' | head -1 | cut -d ':' -f1" % brname)
```

Comment:
Do we know what changed in Ubuntu 20.04? As the '-o' flag should make it stable to use on the CLI

Reply:
Should we use `ip` (iproute2) instead of old tools?

Reply:
bridge is a part of iproute2. The old tool is called 'brctl'

Reply:
there is the trick to use `ls /sys/class/net` to get the exact interface names